dataactbroker/scripts/initialize.py

```diff
@@ -1,20 +1,16 @@
-from dataactvalidator.app import createApp
 from dataactbroker.scripts.setupEmails import setupEmails
```

Comment:
This whole file is being murdered in my PR, so don't worry about editing it any more than this. We don't actually use this initialize.